src/main/java/uk/gov/companieshouse/model/dto/dissolution/DissolutionPatchRequest.java

```diff
@@ -15,6 +15,9 @@
     @JsonProperty("has_approved")
     private boolean hasApproved;
 
+    @JsonProperty("ip_address")
```

Comment:
add the NotBlank annotation and add a test case for this in the controller please

Reply:
still missing

Reply:
@faggc This comment is still outstanding

Reply:
Sorry I don't know how that was removed